stylelint-config.json

```diff
@@ -1,5 +1,59 @@
 {
   "rules": {
+    "color-hex-case": "lower",
```

Comment:
Easier to type and was more common in the existing code.  Also personal preference - much easier to type.